src/FavoritesContext.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState, createContext, useContext } from "react";
+import { useQuoteIndexContext } from "./QuoteIndexContextProvider";
+import { useQuotesContext } from "./QuotesContextProvider";
+
+export const FavoritesContext = createContext(undefined);
+
+export function FavoritesContextProvider({ children }) {
```

Comment:
I don't see a reason why you'd even need this context. We keep information about user's favourite (=liked) quotes in the QuotesContextProvider already. You just need to use the right function to update quotes data in the context (see my comments in the MainPage component as well).